package.json

```diff
@@ -32,7 +35,11 @@
     "concurrently": "^8.2.2",
     "cross-env": "^7.0.3",
     "husky": "^9.0.11",
-    "validate-branch-name": "^1.3.0",
-    "lint-staged": "^15.2.2"
+    "lint-staged": "^15.2.2",
+    "validate-branch-name": "^1.3.0"
+  },
+  "dependencies": {
```

Comment:
Delete argon2 and  react-toastify.

Install react-toastify -> client/
install argon2 -> server/